data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/handler/description/ArtifactDescriptionRequestHandlerSettingsFactory.java

```diff
@@ -12,40 +12,39 @@
  *
  */
 
-package org.eclipse.dataspaceconnector.ids.api.multipart.factory;
+package org.eclipse.dataspaceconnector.ids.api.multipart.handler.description;
 
 import org.eclipse.dataspaceconnector.ids.core.configuration.IllegalSettingException;
 import org.eclipse.dataspaceconnector.ids.core.configuration.SettingResolver;
 import org.jetbrains.annotations.NotNull;
 
-import java.net.URI;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 
-public class DescriptionResponseMessageFactorySettingsFactory {
+public class ArtifactDescriptionRequestHandlerSettingsFactory {
```

Comment:
Unfortunately we can't share the same configuration object with multiple services - that would violate our  [Architecture Key Principles](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/docs/architecture-principles.md#i-fail-fast-and-explicit-configuration).

Reply:
Ok, but defining a `ConnectorIdServiceSettings`/`Factory`/`Results` for the `IdsMultipartApiServiceExtension` will be just one configuration for one service (and multiple handlers), so it will respect the architecture key principles, and it will remove a lot of duplication

Reply:
I don't think that works, as having one configuration object for different services/handlers violates architecture principle I.3
```
Service configuration requirements should always be explicit; as a general rule, do not pass a single configuration object with many values to multiple services. 
```

Reply:
Has been solved during todays refactoring session